docs/recipes/amp-alertmanager-terraform.md

```diff
@@ -0,0 +1,141 @@
+# Terraform as Infrastructure as a Code to deploy Amazon Managed Service for Prometheus and configure Alert manager
+
+In this recipe, we will demonstrate how you can use [Terraform](https://www.terraform.io/) to provision [Amazon Managed Service for Prometheus](https://aws.amazon.com/prometheus/) and configure rules management and alert manager to send notification to a [SNS](https://docs.aws.amazon.com/sns/) topic if a certain condition is met.
+
+
+!!! note
+    This guide will take approximately 30 minutes to complete.
+
+## Prerequisites
+You will need the following to complete the steps in this blog post:
+* [Amazon EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html) 
+* [AWS CLI version 2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+* [Terraform CLI](https://www.terraform.io/downloads)
+* [AWS Distro for Opentelemetry](https://aws.amazon.com/blogs/mt/amazon-managed-service-for-prometheus-is-now-generally-available/)
```

Comment:
Opentelemetry -> OpenTelemetry and let's use https://aws-otel.github.io/ rather than linking to a blog post.

Reply:
modifications made as suggested